src/encoded/schemas/genetic_modification.json

```diff
@@ -92,31 +99,42 @@
             ]
         },
         "category": {
-            "comment": "Insertions need to specify introduced_tags, introduced_sequence, or introduced_gene. Replacements need to specify introduced_sequence. Deletions need either target id, coordinates or deleted sequence specified. Interference needs target id specified and mutagenesis needs treatment specified.",
+            "comment": "Insertions need to specify introduced_tags, introduced_sequence, introduced_gene, or introduced_elements at modified_site_nonspecific. Replacements need to specify introduced_sequence. Deletions need either target id, coordinates or deleted sequence specified. Interference needs target id specified and mutagenesis needs treatment specified. Episome needs to specify introduced_elements.",
             "oneOf": [
                 {
-                    "anyOf": [
+                    "oneOf": [
                         {
-                            "required": ["introduced_tags"],
-                            "properties": {
-                                "category": {
-                                    "enum": ["insertion"]
-                                },
-                                "purpose": {
-                                    "enum": ["tagging"]
+                            "anyOf": [
+                            {
```

Comment:
Need fixing
This is not properly formatted.

Reply:
This branch branches off ENCD-4464-add-GM-category-enums, so that’s not my code. I’ll pull in any changes from that branch if that happens.

Reply:
Got it. I misread the coloring.

Reply:
Could you please explain this formatting issue? And the others alluded to? I'm the one who made the changes.

Reply:
OK. I see this formatting issue now. You mean the { in line 108 and the option after it should have an additional level of indentation.